course_catalog/management/commands/backpopulate_ocw_data.py

```diff
@@ -45,12 +52,15 @@ def handle(self, *args, **options):
                 force_overwrite=options["force_overwrite"],
                 upload_to_s3=options["upload_to_s3"],
                 ignore_flag=True,
+                match_courses=course_id,
             )
             self.stdout.write(
-                "Started task {task} to get ocw course data w/force_overwrite={overwrite}, upload_to_s3={s3}".format(
+                "Started task {task} to get ocw course data "
+                "w/force_overwrite={overwrite}, upload_to_s3={s3}, course_id={course}".format(
                     task=task,
                     overwrite=options["force_overwrite"],
                     s3=options["upload_to_s3"],
+                    course=course_id,
```

Comment:
This should this something like `url_substring` since you are not matching on course id

Reply:
Or maybe `url_contains`